### ISSUE TITLE
Easier option to save worlds to local machine

### ISSUE THREAD

**Issue by JackTheFoxOtter** [Yellow-Dog-Man/Resonite-Issues#3308]
### Is your feature request related to a problem? Please describe.

Currently, to save a world locally, you need to open the world & focus it, then click "Save As / Save Copy", then while the menu is open cycle to your local home with the keyboard shortcuts, then choose "Save Here".

### Describe the solution you'd like

An option to just directly choose "save locally" in the save context menu.

### Describe alternatives you've considered

The above outlined process.

### Additional Context

I've been using locally saved worlds at ITW to show off Cloudscape Harvest without an internet connection, the process for this is pretty fiddly as it stands.

**Comment by epicEaston197** [comment]
This could probably be achieved by having worlds be supported by `resonitepackages` see issue https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/2288 This would allow world creators to sell their world on other sites

**Comment by JackTheFoxOtter** [comment]
No, this has nothing to do with .ResonitePackage files. They are an export / import format, this isn't about exporting but about saving them locally (in the local database, not uploaded to the cloud)

**Comment by MrDj200** [comment]
So the official F.A.Q. mentions being able to store worlds locally. https://wiki.resonite.com/Frequently_Asked_Questions#Can_I_store_things_locally_on_my_hard_drive?

<img width="1092" height="125" alt="Image" src="https://github.com/user-attachments/assets/15afc6ec-6420-40a7-96f6-14c6e9fb8cd0" />


Is the F.A.Q. referring to this fiddly way to save them?
Or is there a proper way nowadays?
